pkg/i18n/japanese.go

```diff
@@ -531,69 +571,77 @@ func japaneseTranslationSet() TranslationSet {
 			UnstageAllFiles:                   "すべてのファイルをアンステージ",
 			StageAllFiles:                     "すべてのファイルをステージ",
 			LcIgnoreExcludeFile:               "ファイルをignore",
+			IgnoreFileErr:                     ".gitignoreをignoreできません",
+			ExcludeFile:                       "ファイルを除外",
+			ExcludeFileErr:                    ".git/info/excludeを除外できません",
+			ExcludeGitIgnoreErr:               ".gitignoreを除外できません",
```

Comment:
```suggestion
			IgnoreFileErr:                     ".gitignore を ignore することはできません",
			ExcludeFile:                       "ファイルを除外",
			ExcludeFileErr:                    ".git/info/exclude を除外することはできません",
			ExcludeGitIgnoreErr:               ".gitignore を除外することはできません",
```

There needs to be a nuance that it is prohibited.